src/ibex/constr/arty-yosys-vivado.xdc

```diff
@@ -0,0 +1,18 @@
+## Based on https://github.com/Digilent/digilent-xdc/blob/master/Arty-A7-100-Master.xdc
```

Comment:
Why is this constraint called arty-yosys-vivado instead of arty.xdc only? I would guess that the same  constraint file would not work on other toolchains. If not, I think this can be removed.

Reply:
Yes, I left option, to use toolchain-specific constraint file, if for some reason, same constraint file won't work on all toolchains, but in this case, it was some leftover. I removed this file and updated ``arty.xdc`` to work on all toolchains